FrontEnd/src/components/ProfilePage/FormComponents/FormFields/ImageField.module.css

```diff
@@ -117,21 +120,88 @@
     padding-top: 5px;
 }
 
-.upload-file__filename--text {
-    color: #1890FF;
-    font-family: "Inter", sans-serif;
+.upload-file__wrapper--banner-page {
+    display: flex;
+    width: 530px;
+    padding-bottom: 24px;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.upload-file__wrapper--banner-card {
+    display: flex;
+    padding-bottom: 24px;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+}
+
+.upload-file__banner-image--title {
+    display: flex;
+    padding-bottom: 8px;
+    align-items: center;
+    gap: 4px;
+    align-self: stretch;
+
+    color: var(--Main-Grey-90, #25292C);
+    font-feature-settings: 'calt' off;
+    font-family: Inter;
     font-size: 14px;
+    font-style: normal;
     font-weight: 400;
     line-height: 22px;
-    letter-spacing: -0.01em;
-    text-align: left;
+    letter-spacing: -0.14px;
 }
 
-.upload-file__filename--trashBin {
-    padding-top: 1px;
+.upload-file__banner-image--page {
+    width: 420px;
+    height: 55.556px;
+    border: 1px solid #C3C3C3;
```

Comment:
`global.css` has not merged so far but can use a variable with a default value

Reply:
I've added this color according to design on Figma . And there is information - this color is not tied to a style or variable in design. So I can't replace it with a variable.